html/elements/input/search.json

```diff
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-search": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input/search",
+            "description": "<code>type=\"search\"</code>",
+            "support": {
+              "chrome": {
+                "version_added": "5"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "4"
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "10.6"
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "5"
+              },
+              "safari_ios": {
+                "version_added": true
+              }
```

Comment:
Could we add `"webview_android"`, too? I think we can safely use the `true` value.

Reply:
I think we can use that